apps/studio/components/interfaces/Organization/TeamSettings/MemberActions.tsx

```diff
@@ -124,62 +112,39 @@ export const MemberActions = ({ member }: MemberActionsProps) => {
     if (!slug) return console.error('Slug is required')
     if (!invitedId) return console.error('Member invited ID is required')
 
-    if (isOptedIntoProjectLevelPermissions) {
-      deleteInvitation(
-        { slug, id: invitedId, skipInvalidation: true },
-        {
-          onSuccess: () => {
-            if (!member.primary_email) return toast.error('Email is required')
-            const projectScopedRole = projectScopedRoles.find((role) => role.id === roleId)
-            if (projectScopedRole !== undefined) {
-              const projects = (projectScopedRole?.project_ids ?? [])
-                .map((id) => allProjects?.find((p) => p.id === id)?.ref)
-                .filter(Boolean) as string[]
-              inviteMember({
-                slug,
-                email: member.primary_email,
-                roleId: projectScopedRole.base_role_id,
-                projects,
-              })
-            } else {
-              inviteMember({ slug, email: member.primary_email, roleId })
-            }
-          },
-        }
-      )
-    } else {
-      deleteInvitationOld(
-        { slug, invitedId, invalidateDetail: false },
-        {
-          onSuccess: () => {
-            inviteMemberOld({
+    deleteInvitation(
+      { slug, id: invitedId, skipInvalidation: true },
+      {
+        onSuccess: () => {
+          if (!member.primary_email) return toast.error('Email is required')
+          const projectScopedRole = projectScopedRoles.find((role) => role.id === roleId)
+          if (projectScopedRole !== undefined) {
+            const projects = (projectScopedRole?.project_ids ?? [])
+              .map((id) => allProjects?.find((p) => p.id === id)?.ref)
+              .filter(Boolean) as string[]
+            inviteMember({
               slug,
-              invitedEmail: member.primary_email!,
-              ownerId: invitedId,
-              roleId,
+              email: member.primary_email,
+              roleId: projectScopedRole.base_role_id,
+              projects,
             })
-          },
-        }
-      )
-    }
+          } else {
+            inviteMember({ slug, email: member.primary_email, roleId })
+          }
+        },
+      }
+    )
   }
 
   const handleRevokeInvitation = (member: OrganizationMember) => {
     const invitedId = member.invited_id
     if (!slug) return console.error('Slug is required')
     if (!invitedId) return console.error('Member invited ID is required')
 
-    if (isOptedIntoProjectLevelPermissions) {
-      deleteInvitation(
-        { slug, id: invitedId },
-        { onSuccess: () => toast.success('Successfully revoked the invitation.') }
-      )
-    } else {
-      deleteInvitationOld(
-        { slug, invitedId },
-        { onSuccess: () => toast.success('Successfully revoked the invitation.') }
-      )
-    }
+    deleteInvitation(
+      { slug, id: invitedId },
+      { onSuccess: () => toast.success('Successfully revoked the invitation.') }
```

Comment:
might need to double check - on my somehow clicking "Cancel invite" isn't showing this success toast

Reply:
Worked for me on this UI.